pages/establishment/dashboard/views/index.jsx

```diff
@@ -49,11 +50,22 @@ const Index = ({
           <PanelList
             panels={links.filter(link => allowedActions.includes(link.permissions)).map((link, index) => <DashboardLink key={index} { ...link } />)}
           />
+          {
+            establishment.status !== 'active' && allowedActions.includes('establishment.update') && !openApplication &&
```

Comment:
This is a lot of conditions to have inline. Could we pull it out into a `const canApply = establishment.status !== 'active' && allowedActions.includes('establishment.update') && !openApplication;` ?